.pre-commit-config.yaml

```diff
@@ -21,7 +21,7 @@ repos:
     rev: v0.0.269
     hooks:
       - id: ruff
-        args: ["--fix", "--fixable=I001", "--exit-non-zero-on-fix"]
+        args: ["--fix", "--fixable=I001,F401", "--exit-non-zero-on-fix"]
```

Comment:
Can we only use `pyproject.toml`?

Reply:
Sure. I could move `fixable` to `pyproject.toml`. The downside is that this would also apply when a developer runs explicit `ruff --fix` locally for other fixes that we haven't explicitly whitelisted.

Or we could get rid of `fixable` entirely, which would apply all fixes in selected categories (`select = [...]`)

For my own projects I've adopted a more cautious approach and explicitly named the fixable rules. But I haven't seen any issues so far with the autofixes.

What do you think?

Reply:
Sure, I've removed `--fixable` then, which applies all fixes (when used with `--fix`).